Sources/StreamChat/Workers/ChannelListCleanupUpdater_Tests.swift

```diff
@@ -0,0 +1,116 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChat
+@testable import StreamChatTestTools
+import XCTest
+
+final class ChannelListCleanupUpdater_Tests: StressTestCase {
+    typealias ExtraData = NoExtraData
+        
+    var database: DatabaseContainerMock!
+    var webSocketClient: WebSocketClientMock!
+    var apiClient: APIClientMock!
+    
+    var channelListCleanupUpdater: ChannelListCleanupUpdater<ExtraData>?
+    var channelListUpdater: ChannelListUpdaterMock<NoExtraData>!
+    
+    override func setUp() {
+        super.setUp()
+        
+        database = DatabaseContainerMock()
+        webSocketClient = WebSocketClientMock()
+        apiClient = APIClientMock()
+        channelListUpdater = ChannelListUpdaterMock(database: database, apiClient: apiClient)
+        
+        channelListCleanupUpdater = ChannelListCleanupUpdater(
+            database: database,
+            apiClient: apiClient,
+            channelListUpdater: channelListUpdater
+        )
+    }
+    
+    override func tearDown() {
+        apiClient.cleanUp()
+        
+        AssertAsync {
+            Assert.canBeReleased(&channelListCleanupUpdater)
+            Assert.canBeReleased(&database)
+            Assert.canBeReleased(&webSocketClient)
+            Assert.canBeReleased(&apiClient)
+        }
+        
+        super.tearDown()
+    }
+    
+    func test_cleanupChannelList_cleansChannelsData() throws {
+        let cid1 = ChannelId.unique
+        let cid2 = ChannelId.unique
+        
+        try database.createChannel(
+            cid: cid1,
+            withMessages: true,
+            withQuery: true,
+            hiddenAt: Date(timeIntervalSince1970: 56789),
```

Comment:
why not `.unique`?

Reply:
Data in unit tests should be static unless it's the idea behind a stress test to run it with various random inputs

Reply:
Yes, that's exactly the idea :)